root/static/scripts/release/components/ReleaseRelationshipEditor.js

```diff
@@ -1481,8 +1487,20 @@ const TrackRelationshipsSection = React.memo(({
             }
             dispatch={dispatch}
             recordingSelectionCount={recordingCount}
+            releaseHasUnloadedTracks={releaseHasUnloadedTracks}
             workSelectionCount={workCount}
           />
+          {releaseHasUnloadedTracks ? (
+            <div className="form-help">
+              <p>
+                {l(
+                  `Some tracks/mediums haven’t been loaded yet.
+                   If you want to make batch operations on all tracks,
+                   please fully load all mediums beforehand.`,
```

Comment:
Attempted improvements:
- Locate things involved in “batch operations”
- Loading is needed even to select “some mediums” not just “all tracks”

```suggestion
                  `Some tracks/mediums haven’t been loaded yet.
                   If you want to select some mediums below
                   in order to use with the batch-editing tools above,
                   please load all the tracks from these mediums beforehand.`,
```